com.github.Bleuzen.FFaudioConverter.json

```diff
@@ -0,0 +1,79 @@
+{
+    "app-id": "com.github.Bleuzen.FFaudioConverter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "rename-icon": "ffaudioconverter",
+    "rename-desktop-file": "ffaudioconverter.desktop",
```

Comment:
You should rename the desktop and icon files upstream.

Reply:
Why and to what?
How I understood it is that this are the original names and flatpak renames them to have the app id in their name. So sorry, I'm new to flatpak, but where is the problem?

Reply:
These flatpak options are hacks to make it easier to package third party packages. Since you are upstream they should just be renamed there.

The freedesktop specs recommend this style of naming:

- https://standards.freedesktop.org/desktop-entry-spec/latest/ar01s02.html
- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html